src/redux/books/books.js

```diff
@@ -0,0 +1,30 @@
+const ADD_BOOK = 'bookstore/books/addBook';
+const REMOVE_BOOK = 'bookstore/books/removeBook';
+const lastId = 0;
+export default function addRemoveReducer(state = [], action) {
+  switch (action.type) {
+    // add action
+    case ADD_BOOK: return [
+      ...state, {
+        id: lastId + 1,
+        title: action.title,
+        author: action.author,
+      },
+    ];
```

Comment:
- [ ] **[OPTIONAL]** If you implemented my optional change above, then we can refactor this to something like so:

  ```
  case ADD_BOOK:
    return [
      ...state, action.payload
    ];
  ```